src/app/components/forms/game/video/video.ts

```diff
@@ -29,8 +30,8 @@ export default class FormGameVideo extends BaseForm<FormModel> implements FormOn
 	get videoData() {
 		const url = this.formModel._url;
 		if (url) {
-			const youtubeMatch = url.match(GameVideo.REGEX.YOUTUBE);
-			const vimeoMatch = url.match(GameVideo.REGEX.VIMEO);
+			const youtubeMatch = url.match(REGEX_YOUTUBE);
```

Comment:
I feel like we can put that getYoutubeVideoId function somewhere in util folder and then change this code to use a generic function that would get the video match with the "provider". What do you think?